scala/scala_import.bzl

```diff
@@ -1,114 +1,49 @@
-#intellij part is tested manually, tread lightly when changing there
-#if you change make sure to manually re-import an intellij project and see imports
-#are resolved (not red) and clickable
```

Comment:
why did we delete the comment here? Seems like it is still relevant.

Reply:
Overlooked it while porting this code over. I'll add it back (and actually prove out that this works for IntelliJ).

Reply:
Comment added back in the body of the method.